src/main/java/org/postgresql/sql2/communication/packets/parsers/BinaryParser.java

```diff
@@ -3,275 +3,275 @@
   }
 
-  public static Object charsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object charsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object namesend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object namesend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object int8send(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object int8send(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object int2send(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object int2send(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object int2vectorsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object int2vectorsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object int4send(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
-    return BinaryHelper.readInt(bytes[start], bytes[start + 1], bytes[start + 2], bytes[start + 3]);
+  public static Object int4send(byte[] bytes, Class<?> requestedClass) {
+    return BinaryHelper.readInt(bytes[0], bytes[1], bytes[2], bytes[3]);
   }
 
-  public static Object regprocsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object regprocsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object oidsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object oidsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object tidsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object tidsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object xidsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object xidsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object cidsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object cidsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object oidvectorsend(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object oidvectorsend(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object pg_ddl_command_send(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object pg_ddl_command_send(byte[] bytes, Class<?> requestedClass) {
     return null;
   }
 
-  public static Object json_send(byte[] bytes, Integer start, Integer end, Class<?> requestedClass) {
+  public static Object json_send(byte[] bytes, Class<?> requestedClass) {
```

Comment:
I'm not gonna do a code review of this whole thing, but lots of scary things like underscores in identifiers (and even then, it's inconsistent as above they aren't there). I think a very close review of every character typed is warranted for these types of commits (I am not associated with this project nor do I speak for it, just an interested observer).